tests/test_core.py

```diff
@@ -1411,6 +1431,8 @@ def test_exceptions_3(self):
     self.do_run(src, 'Caught exception: Hello\nDone.', ['2'], no_build=True)
 
   def test_exceptions_white_list(self):
+    if self.is_wasm_backend():
```

Comment:
Oh sorry, that's not quite right; the EH tests are disabled in bulk inside of the `do_run` implementation (so we don't have to use the decorator on every EH test). But because this test does other things, it does need the decorator.